packages/notifier-seeder/src/index.ts

```diff
@@ -1,27 +1,78 @@
-import { EachMessagePayload } from "kafkajs";
+/* eslint-disable functional/immutable-data */
 import { runConsumer } from "kafka-iam-auth";
+import { EachMessagePayload } from "kafkajs";
 import {
-  kafkaConsumerConfig,
+  CatalogTopicConfig,
   catalogTopicConfig,
+  getContext,
+  kafkaConsumerConfig,
   logger,
+  loggerConfig,
+  messageDecoderSupplier,
+  notificationConfig,
 } from "pagopa-interop-commons";
+import { v4 as uuidv4 } from "uuid";
+import { kafkaMessageProcessError } from "pagopa-interop-models";
+import { convertToEserviceV1 } from "./models/catalogEventV2ConverterToV1.js";
+import { buildCatalogMessage } from "./models/catalogEventMessage.js";
+import { initQueueManager } from "./queue-manager/queueManager.js";
 
 const config = kafkaConsumerConfig();
-const topics = catalogTopicConfig();
+const topicsConfig = catalogTopicConfig();
+const logConfig = loggerConfig();
+const queueConfig = notificationConfig();
+const queueManager = initQueueManager({
+  queueUrl: queueConfig.queueUrl,
+  messageGroupId: "message_group_all_notification",
+  logLevel: logConfig.logLevel,
+});
+
+function processMessage(topicConfig: CatalogTopicConfig) {
+  return async (kafkaMessage: EachMessagePayload): Promise<void> => {
+    try {
+      /* 
+        TODO: handle correlationId from message when 
+        PR https://github.com/pagopa/interop-be-monorepo/pull/310 is merged 
```

Comment:
Could you either rebase on that PR or track this as a further task on Jira? So that we don't lose track of it, it's part of https://pagopa.atlassian.net/browse/IMN-301 so it's required for the release of catalog

Reply:
Now it reads correlation ID from message